contrib/pyln-testing/pyln/testing/utils.py

```diff
@@ -973,7 +973,7 @@ def restart(self, timeout=10, clean=True):
         self.start()
 
     def fundchannel(self, l2, amount=FUNDAMOUNT, wait_for_active=True,
-                    announce_channel=True, **kwargs):
+                    announce_channel=True, connect=False, **kwargs):
```

Comment:
I agree to have `connect=True` as default, but since I added this as a new optional parameter and the function did no connection prior to this, my initial intention was not to change the default logic of the function.

Let me try to change it and see if the other tests are all still green.

Reply:
If we default to `True`, we may also simply remove that optional parameter and connect (if not connected) as for the other functions `openchannel`...